nextflow.config

```diff
@@ -1,30 +1,89 @@
 manifest {
-    mainScript = 'momofy.nf'
+    mainScript = 'main.nf'
     nextflowVersion = '>=21.10.0'
 }
 
 params {
-	// MoMofy params
-    	max_cores = Runtime.runtime.availableProcessors()
-    	cores = Runtime.runtime.availableProcessors().intdiv(4)
-    	memory = '12'
+    max_cores = Runtime.runtime.availableProcessors()
+    cores = Runtime.runtime.availableProcessors().intdiv(4)
+    memory = '12'
 
-	help = false
-	profile = false
+    help = false
+    profile = false
 
-	assembly = ''
-	outdir = 'MoMofy_results'
-	user_genes = false
-	palidis = false
-	gff_validation = true
-
-	mobileog_db="${projectDir}/databases/mobileOG_beatrix1.6.dmnd"
+    // Inputs
+    assembly = ''
+    outdir = 'mobilome_results'
+    user_genes = false
+    palidis = false
+    virify = false
+    gff_validation = true
 
+    // Databases
+    mobileog_db = ''
+    amrfinder_plus_db = ''
 }
 
-singularity {
-	enabled = true
-	autoMounts = true
-	cacheDir = "${projectDir}/singularity"
+profiles {
+    //executors
+    local {
+        executor {
+            name = "local"
+            cpus = params.max_cores
+        } 
+        includeConfig 'config/local.config'
+    }
+
+    lsf {
+        executor {
+            name = "lsf"
+            queue = 'production'
```

Comment:
the queue name is HPC dependent

Reply:
Ok, I have moved that bit into the ebi profile :D